src/main.py

```diff
@@ -0,0 +1,102 @@
+from AlertsAggregator import AlertsAggregator
+from OrefAlertsRetrieve import OrefAlertsRetrieveData  
+from fastapi import FastAPI, HTTPException 
+import pandas as pd
+import requests
+from Errors import RetrieveDataException, WrongSettlementException
+from models import AlertsQueryInput
+from datetime import datetime, timedelta
+app = FastAPI()
+
+CSV_FILE_PATH = "C:/Projects/Python/Projects/BestTimeToShower/data/data.csv"
+dataframe = pd.read_csv(CSV_FILE_PATH)
+getdata = OrefAlertsRetrieveData()
+alert_aggregator = AlertsAggregator(dataframe)
```

Comment:
what happends if I call get_oref_alert??
a new csv file will be created but AlertsAggregator was already created with an old dataframe